### ISSUE TITLE
Machinomy initialises storage every method call

### ISSUE THREAD

**Issue by ukstv** [machinomy/machinomy#106]
File that gives an error: https://gist.github.com/ukstv/bb2915784dfe7102eca1c599018f0aaa

It results in
```
Error: ENOENT: no such file or directory, rename 'machinomy~' -> 'machinomy'
```

Reason: Machinomy tries to open few instances of storage with every method call, like https://github.com/machinomy/machinomy/blob/master/index.ts#L134, https://github.com/machinomy/machinomy/blob/master/index.ts#L157 and so on.

Related NeDB bug report: https://github.com/louischatriot/nedb/issues/320

Proposed fix: initialise storage once per instance.

CC @djKooks

**Comment by burtovoy** [comment]
The problem is fixed in the 1.4.0 version. 
But the example from the gist  is incorrect anyway.  
Machinomy.buy doesn't return hash with channel id. It returns a promise. 
Example: https://github.com/machinomy/machinomy/blob/master/examples/client_es5.js#L26

**Comment by djKooks** [comment]
@burtovoy Okay thanks! I'll try.
